<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshDock</class>
 <widget class="QDockWidget" name="MeshDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QComboBox" name="meshComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbConnectors">
      <property name="title">
       <string>Connectors</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="meshConnectors"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
